From 202176ddd60d709372bd4dd05fd5d6490fc2559e Mon Sep 17 00:00:00 2001
From: Gabor Abonyi <gabor.abonyi@arm.com>
Date: Mon, 22 Jan 2024 14:55:03 +0100
Subject: [PATCH 1/2] resources: Replace model wav2letter with tiny_wav2letter

`wav2letter` model needs singificant amount of memory (around 14Mb),
where as `tiny_wav2letter` needs around 4Mb.
Update the location of the resource where referenced for
the `asr` and `kws_asr` use cases.

Signed-off-by: Gabor Abonyi <gabor.abonyi@arm.com>
---
 scripts/py/use_case_resources.json | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/scripts/py/use_case_resources.json b/scripts/py/use_case_resources.json
index 80fa28d..9e50fb6 100644
--- a/scripts/py/use_case_resources.json
+++ b/scripts/py/use_case_resources.json
@@ -16,16 +16,16 @@
   {
     "name": "asr",
     "url_prefix": [
-      "https://github.com/ARM-software/ML-zoo/raw/1a92aa08c0de49a7304e0a7f3f59df6f4fd33ac8/models/speech_recognition/wav2letter/tflite_pruned_int8/"
+      "https://github.com/ARM-software/ML-zoo/raw/eb2170aac1317f00b128ab82bdb159cfcca36ea6/models/speech_recognition/tiny_wav2letter/tflite_pruned_int8/"
     ],
     "resources": [
       {
-        "name": "wav2letter_pruned_int8.tflite",
-        "url": "{url_prefix:0}wav2letter_pruned_int8.tflite"
+        "name": "tiny_wav2letter_pruned_int8.tflite",
+        "url": "{url_prefix:0}tiny_wav2letter_pruned_int8.tflite"
       },
       {
         "name": "ifm0.npy",
-        "url": "{url_prefix:0}testing_input/input_2_int8/0.npy"
+        "url": "{url_prefix:0}testing_input/input_1_int8/0.npy"
       },
       {
         "name": "ofm0.npy",
@@ -96,18 +96,18 @@
   {
     "name": "kws_asr",
     "url_prefix": [
-      "https://github.com/ARM-software/ML-zoo/raw/1a92aa08c0de49a7304e0a7f3f59df6f4fd33ac8/models/speech_recognition/wav2letter/tflite_pruned_int8/",
+      "https://github.com/ARM-software/ML-zoo/raw/eb2170aac1317f00b128ab82bdb159cfcca36ea6/models/speech_recognition/tiny_wav2letter/tflite_pruned_int8/",
       "https://github.com/ARM-software/ML-zoo/raw/9f506fe52b39df545f0e6c5ff9223f671bc5ae00/models/keyword_spotting/micronet_medium/tflite_int8/"
     ],
     "resources": [
       {
-        "name": "wav2letter_pruned_int8.tflite",
-        "url": "{url_prefix:0}wav2letter_pruned_int8.tflite"
+        "name": "tiny_wav2letter_pruned_int8.tflite",
+        "url": "{url_prefix:0}tiny_wav2letter_pruned_int8.tflite"
       },
       {
         "sub_folder": "asr",
         "name": "ifm0.npy",
-        "url": "{url_prefix:0}testing_input/input_2_int8/0.npy"
+        "url": "{url_prefix:0}testing_input/input_1_int8/0.npy"
       },
       {
         "sub_folder": "asr",
-- 
2.40.1

